fastlane/Fastfile

```diff
@@ -1324,16 +1326,27 @@ end
 # Kicks off a Buildkite build
 # -----------------------------------------------------------------------------------
 def trigger_buildkite_release_build(branch:, beta:)
-  build_url = buildkite_trigger_build(
-    buildkite_organization: 'automattic',
-    buildkite_pipeline: 'woocommerce-ios',
-    branch: branch,
-    environment: { BUILDKITE_BETA_RELEASE: beta },
-    pipeline_file: 'release-builds.yml'
-  )
+  beta = beta.to_s
+  environment = { 'BUILDKITE_BETA_RELEASE' => beta }
```

Comment:
I'm surprised we need to use the `BUILDKITE_` prefix 🤔

Reply:
Yeah, we need this to be propagated to the macOS VM, so if I understand it correctly we still need it.

Reply:
> so if I understand it correctly we still need it.

Not if we adopt the alternative approach that we've been adopting in all other repos (resolve env var in `.yml` pass resolved value as parameter to invoked `.sh`), to avoid using env vars that are supposed to be reserved for Buildkite internals only 😉 

See https://github.com/Automattic/hostmgr/issues/110 for more details (and the "Workaround in the meantime" section)

Reply:
Yeah, I later realized it's already the case (so not sure why it ever needed the `BUILDKITE_` prefix) and updated the comment. It's resolved and used only in the `.yml`, so I updated it on cbe3b4681a90e8c2f43a386c569c8153ef98da4e.